src/Components/Routes/Routes.js

```diff
@@ -1,28 +1,27 @@
 import React from "react";
-import { Route, Router, Switch } from "react-router-dom";
+import { Route } from "react-router-dom";
```

Comment:
Note to remove unused variables in the future as it will interfere with continuous integration 🐲

Reply:
I second Catlyns comment. Good to know.